### ISSUE TITLE
FortiOS 7.2.1 supported？

### ISSUE THREAD

**Issue by tqangxl** [bluecmd/fortigate_exporter#191]
FortiOS for fotigate 7.2.1 supported？

**Comment by JackKeeley** [comment]
I recently upgraded one of our FortiGate 60E firewalls from 7.0.5 to 7.2.1 - so far the only issues that we have had so far is higher than average CPU usage (only happens when polling the API with the exporter), and issues with the forgiate not responding to API requests after rebooting (normally delayed by ~10minutes) I have attached an example of our CPU usage on 7.0.5 vs 7.2.1 if anyone wants it to reference. 
![60e](https://user-images.githubusercontent.com/5621468/189808394-009b4c2d-0e75-473b-be16-a59e2b732f6e.PNG)

**Comment by p-v-a** [comment]
@JackKeeley, is it fortigate cluster? I ran into similar issue during failover to second node in the cluster (reboot causes failover as well). The root cause looks like to be TCP session handover during failover, while TLS or something higher up in TCP stack can't handle this, so new node end up ignoring exporters packets until HTTP timeout kills session on exporter end. 

I have solved this but just adding environmental variable to my exporter startup that disables http2. not an ideal, but... see [issue 220](https://github.com/bluecmd/fortigate_exporter/issues/220).
